Makefile.tools

```diff
@@ -188,8 +188,8 @@ $(SHELLCHECK) shellcheck: $(TOOLS_DIR)
 		set -ex ;\
 		[[ -f $(SHELLCHECK) ]] && exit 0 ;\
 		cd $$(mktemp -d) ;\
-		OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-		curl -sSLo shellcheck-stable.tar.xz "https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.$$(OS).$$(ARCH).tar.xz";\
+		OS=$(shell go env GOOS) && ARCH=$(shell uname -m) && \
```

Comment:
I don't think that we need to change the way the ARCH var is filled?

Reply:
output of arch is:
`ARCH=amd64`
then:
`https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.limux.amd64.tar.xz`
and that's 404

Correct one is:
`https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.linux.x86_64.tar.xz`

Reply:
:+1: